### PR TITLE
First pass at using folderfun

### PR DESCRIPTION
**Pull request by vreuter**
Not sure this is ready for merge but am opening mostly to see if this is on the right track, @nsheff

### CODE REVIEW

**Review by stolarczyk — CHANGES_REQUESTED** [review]
I've deleted the `inff` function as suggested in https://github.com/databio/folderfun/issues/4. We'll need to use the `setff`, then.

**Comment by vreuter** [comment]
:ok_hand: thanks @michalstolarczyk

**Comment by stolarczyk** [comment]
```r
.getVar = function(name) {
  opt = getOption(name)
  if(!is.null(opt))
    return(opt)
  env = Sys.getenv(name)
  if(!identical("",env))
    return(env)
  return(NULL)
}

folderfun::setff("Proc",.getVar("PROCESSED.PROJECT"))
```

For example, this is how we could do it

File: DESCRIPTION [hunk 1]
```diff
@@ -10,6 +10,10 @@ Description: Makes it easy to initialize a customized workspace, relying on
 	shell environment variables to point easy function aliases so you can control
 	project-specific results directories, data inputs, shared temporary files, and
 	other resources
+Remotes:
+    databio/folderfun
```
Comment (nsheff) [inline DESCRIPTION]: can take this out soon... I've submitted folderfun to CRAN

File: R/dirAliases.R [hunk 2]
```diff
@@ -20,65 +20,35 @@ dirOut = function(...) {
 }
 
 # output dir without any subdir.
-dirOutRoot = function(...) {
-	dirWrap("ROUT.DIR", ...)
-}
+dirOutRoot = function(...) ffROUT.DIR(...)
```
Comment (nsheff) [inline R/dirAliases.R:23]: instead of this, let's just do:

`setff("OutRoot", "ROUT.DIR")`

and then use `ff` instead of `dir` from now on...

**Comment by nsheff** [comment]
I'm going to merge this in and make some changes now...

**Pull request merged**